src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2372,11 +2376,8 @@ private boolean isFeedbackSessionViewableTo(
         }
```

Comment:
Isn't this logic different from the original? In the original, private sessions are visible to the creator only. In this version, instructors can view private sessions of other instructors in the course. If that is the case, then you introduced a bug which should have been caught by a test.